x-pack/plugins/apm/public/components/app/onboarding/commands/django.ts

```diff
@@ -34,9 +32,6 @@ export const django = `INSTALLED_APPS = (
 )
 
 ELASTIC_APM = {
-  # {{serviceNameHint}}
-  'SERVICE_NAME': 'my-service-name',
```

Comment:
We still want to show the service name in the code snippet. It's just the boilerplate in the table that doesn't make sense.